src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -881,6 +889,19 @@ private static Conversion ClassifyNullLiteralConversion(BoundExpression source,
             return Conversion.NoConversion;
         }
 
+        private Conversion ClassifyDefaultConversion(BoundDefaultPendingInference source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            Debug.Assert((object)source != null);
+            Debug.Assert((object)destination != null);
+
+            if (destination.IsStructType() && !destination.IsNullableType())
+            {
```

Comment:
From our last discussion, I thought it may be good to avoid ambiguity between `default` and `null`.
The thread on github shows interest to allow `default` anywhere `default(type)` was allowed.
I listed open questions for LDM in #13602